### DISCUSSION TITLE
Total Number of Domains DNS blocklists

### DISCUSSION THREAD

**Discussion by AxReds** [AdguardTeam/AdGuardHome#6832] [Ideas]
A counter in the homepage dashboard that display the sum of all unique domains blocked by the DNS blocklist such as what pi-hole does with this counter.

![image](https://github.com/AdguardTeam/AdGuardHome/assets/72096203/eaf901fc-5fbf-41e6-ba1e-73762aeab743)

**Comment by ghost** [comment]
You can make that feature request [here](https://github.com/AdguardTeam/AdGuardHome/issues/new?assignees=&labels=feature+request&projects=&template=feature.yml).
